docs/style.css

```diff
@@ -195,11 +195,9 @@ table.attributes td:nth-last-child(2) {
     text-transform: uppercase;
     background: #ffee99;
     color: #b39500; }
-  table.attributes td:nth-last-child(2):empty {
+  table.attributes td:nth-last-child(2).x {
```

Comment:
"x" is not a very understandable name for a class. I'd suggest something more meaningful, like "empty" or "no" or whatever else you think makes sense :)